### PR TITLE
Normalize the config file to minimize noise later

### PR DESCRIPTION
**Pull request by kytrinyx**
We will be scripting a number of changes to the track configuration.
This runs configlet fmt, which ensures that any further changes can be minimized.

See https://github.com/exercism/meta/issues/95

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -1,909 +1,741 @@
...
     {
-      "uuid": "4fe19484-4414-471b-a106-73c776c61388",
-      "slug": "hello-world",
       "core": false,
-      "unlocked_by": null,
       "difficulty": 1,
-      "topics": [
-
-      ]
+      "slug": "hello-world",
```
Comment (Insti) [inline config.json]: slug should be the first key in the exercises hash.

File: config.json [hunk 2]
```diff
@@ -1,909 +1,741 @@
...
+      "topics": null,
+      "unlocked_by": null,
```
Comment (Insti) [inline config.json]: core and unlocked_by should be co-located.

File: config.json [hunk 3]
```diff
@@ -1,909 +1,741 @@
...
-  "gitter": "ruby",
-  "solution_pattern": "[Ee]xample|\\.meta/solutions/[^/]*\\.rb",
+  "blurb": "Ruby is a dynamic, open source programming language with a focus on simplicity and productivity. It has an elegant syntax that is natural to read and easy to write.",
   "exercises": [
     {
-      "uuid": "4fe19484-4414-471b-a106-73c776c61388",
-      "slug": "hello-world",
       "core": false,
-      "unlocked_by": null,
       "difficulty": 1,
```
Comment (Insti) [inline config.json]: difficulty and topics should be co-located

File: config.json [hunk 4]
```diff
@@ -1,909 +1,741 @@
 {
-  "language": "Ruby",
```
Comment (Insti) [inline config.json]: language should remain the first key.

File: config.json [hunk 5]
```diff
@@ -1,909 +1,741 @@
...
     }
   ],
-  "foregone": [
-
-  ]
-}
+  "foregone": [],
+  "gitter": "ruby",
+  "language": "Ruby",
+  "solution_pattern": "[Ee]xample|\\.meta/solutions/[^/]*\\.rb"
```
Comment (Insti) [inline config.json]: These 3 should be at the top with the other one line entries.

File: config/maintainers.json [hunk 6]
```diff
@@ -1,64 +1,64 @@
 {
   "maintainers": [
     {
-      "github_username": "kotp",
-      "show_on_website": false,
```
Comment (Insti) [inline config/maintainers.json]: These two are the most important and should remain in this order at the beginning.

File: config.json (hunk 4 above)
Comment (Insti) [reply config.json]: Until track_id comes along..

**Comment by kytrinyx** [comment]
There's a discussion happening in exercism/meta#95 about how this normalization degrades the readability of the configs.

That discussion has some good suggestions for improvements to the serializer.
I'd appreciate it if you would weigh in on that discussion with any suggestions for improving how we handle configs, consistency, and normalization.

Meanwhile, I'm going to close this and wait for that discussion to play out.

I'm sorry about the noise, and any extra work that I've caused you.

**Pull request closed without merging**